packages/frenetic/frenetic.5.0.5/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+synopsis: "The Frenetic Programming Language and Runtime System"
+maintainer: "Nate Foster <jnfoster@cs.cornell.edu>"
+authors: "Arjun Guha <arjun@cs.umass.edu>, Nate Foster <jnfoster@cs.cornell.edu>, Steffen Smolka <smolka@cs.cornell.edu>"
+homepage: "http://frenetic-lang.org"
+license: "LGPL-3.0-or-later"
+bug-reports: "https://github.com/frenetic-lang/frenetic/issues"
+dev-repo: "git+https://github.com/frenetic-lang/frenetic.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+]
+depends: [
+  "ocaml" {build & >= "4.11.0"}
+  "dune" {build & >= "2.0"}
+  "odoc"
+  #########################
+  "async"  {>= "v0.14.0" }
+  "base64" {>= "3.5.0"}
+  "cohttp" {>= "4.0.0"}
+  "cohttp-async" {>= "4.0.0" }
+  "core"   {>= "v0.14.0" }
+  "cstruct" {>= "6.0.1"}
+  "cstruct-sexp" {>= "6.0.1"}
+  "ipaddr" {>= "5.2.0"}
+  "menhir" {>= "20210419"}
+  "mparser" {>= "1.3"}
+  "mparser-re" {>= "1.3"}
+  "ocamlgraph" {>= "2.0.0"}
+  "open" {>= "0.2.1"}
+  "ppxlib" {>= "0.22.0"}
+  "ppx_jane" {>= "0.14.0"}
+  "ppx_cstruct" {>= "5.2.0"}
+  "ppx_deriving" {>= "5.1"}
+  "sedlex" {>= "2.2" & <= "2.3"}
+  "sexplib" {>= "0.14.0"}
```

Comment:
```suggestion
  "sexplib" {>= "v0.14.0"}
```
The `v` prefix is required on Janestreet packages

Reply:
Fixed, thanks!